scantool/diag.h

```diff
@@ -138,7 +138,7 @@ typedef uint16_t flag_type;	//this is used for L2 type flags (see diag_l2.h)
  * Only applicable if DIAG_L1_DOESKEEPALIVE is set
  *
  * data = (struct diag_msg *).
- * The (struct diag_msg)->data member must be a raw message (including headers, checksums etc)
+ * The (struct diag_msg)->data member must be a raw message (including headers)
  *
```

Comment:
The convention for diag_l1_send() is that the L2 supplies the checksum only if DIAG_L1_DOESL2CKSUM isn't set. I think we should follow that for DIAG_IOCTL_SETWM also.